### PR TITLE
Andela Day 1 challenge

### PR DESCRIPTION
**Pull request by nuwagalb**
Adds the Andela Day One challenge requirements to my master branch

### CODE REVIEW

File: UI/add_list.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                <input type="text" name="description" class="form-field">
+                <br>
+                
+                <!--Submit field-->
+                <input type="submit" value="Create" class="submit" id="add_list_submit">
+
+            </form>
+        </div>
+    </body>
+</html>
```
Comment (redondo3) [inline UI/add_list.html]: try to leave some space after your </html> closing tag

**Comment by nuwagalb** [comment]
Just use Balsamiq mockups. Let me also update my readme.md file to address this specific issue. Otherwise should i merge the changes according to what you've seen

**Comment by redondo3** [comment]
Go ahead and merge. All looks well

**Comment by nuwagalb** [comment]
Thanks for your input.

**Pull request merged**